packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -442,22 +442,35 @@ function TableVirtualizer({layout, collection, focusedKey, renderView, renderWra
     transitionDuration
   });
 
+  let scrollToItem = useCallback((key) => {
```

Comment:
I know this wasn't previous in the aria package, but is there a reason it isn't? I could see someone wanting this.

Reply:
This had some values/things specific to our tableview implementation (scroll duration of 0, and handling due to how we make the checkbox column sticky).